doc/nrf/ug_nrf52_developing.rst

```diff
@@ -0,0 +1,260 @@
+
+.. _build_pgm_nrf52:
+
+Building and programming
+************************
+
+You can program applications and samples on the nRF9160 DK after obtaining the corresponding firmware images.
+
+To program applications using the Programmer app from `nRF Connect for Desktop`_, follow the instructions in :ref:`nrf52_gs_installing_application`.
+In Step 5, choose the :file:`.hex` file for the application you are programming.
+
+.. _build_pgm_nrf52_vsc:
+
+Building and programming using |VSC|
+====================================
+
+|vsc_extension_instructions|
+
+Complete the following steps after installing the |nRFVSC|:
+
+.. |sample_path_vsc| replace:: :file:`nrf/applications/nrf_desktop`
+
+.. |vsc_sample_board_target_line| replace:: see the :ref:`list of supported boards <nrf52_supported_boards>` for the build target corresponding to the nRF52 Series DK you are using.
+
+.. include:: /includes/vsc_build_and_run.txt
+
+#. Connect the nRF52 Series DK to the computer with a micro-USB cable, and then turn on the DK.
+
+   **LED1** starts blinking.
+
+#. In |VSC|, click the :guilabel:`Flash` option in the :guilabel:`Actions View`.
+
+   If you have multiple boards connected, you are prompted to pick a device at the top of the screen.
+
+   A small notification banner appears in the bottom right corner of |VSC| to display the progress and confirm when the flash is complete.
+
+.. _build_pgm_nrf52_cmdline:
+
+Building and programming on the command line
+============================================
+
+Complete the :ref:`command-line build setup <build_environment_cli>` before you start building |NCS| projects on the command line.
+
+To build and program the source code from the command line, complete the following steps:
+
+1. Open a terminal window.
+#. Go to the specific sample or application directory.
+   For example, the folder path is ``ncs/nrf/applications/nrf_desktop`` when building the source code for the :ref:`nrf_desktop` application.
+
+#. Make sure that you have the required version of the |NCS| repository by pulling the |NCS| repository, `sdk-nrf`_ on GitHub using the procedures described in :ref:`dm-wf-get-ncs` and :ref:`dm-wf-update-ncs`.
+
+#. To get the rest of the dependencies, run the ``west update`` command as follows:
+
+   .. code-block:: console
+
+      west update
+
+#. To build the sample or application code, run the ``west build`` command as follows:
+
+   .. parsed-literal::
+      :class: highlight
+
+      west build -b *build_target* -d *destination_directory_name*
+
+   For the *build_target* parameter, see the :ref:`list of supported boards <nrf52_supported_boards>` for the build target corresponding to the nRF52 Series DK you are using.
+
+   .. note::
+
+	   The parameter *destination_directory_name* can be used to optionally specify the destination directory in the west command.
+	   Unless a *destination_directory_name* is specified, the build files are automatically generated in ``build/zephyr/``.
```

Comment:
```suggestion
	   The build files are automatically generated in ``build/zephyr/`` if you have not specified a *destination_directory_name* .
```

Reply:
Changed "automatically generated in ``build/zephyr/``" to "generated in ``build/zephyr/`` by default"